pkgs/by-name/sl/slackdump/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  buildGoModule,
+  lib,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "slackdump";
+  version = "2.5.11";
+
+  src = fetchFromGitHub {
+    owner = "rusq";
+    repo = "slackdump";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-othSIR+KyekUt+/LWhaM5Y6LGsujDCZTfLJotTcPXc0=";
+  };
+
+  vendorHash = "sha256-6GVXzaJ5Lrt4fF0c9cbx6AqpgCwlYgKf6jUJkUyxc0s=";
+
+  meta = {
+    homepage = "https://github.com/rusq/slackdump";
+    changelog = "https://github.com/rusq/slackdump/releases/tag/v${version}";
+    description = "Tools for saving Slack's data without admin privileges";
+    platforms = lib.platforms.unix;
```

Comment:
`lib.platforms.unix` is actually more than go supports, can be removed so it defaults to  `go.meta.platforms`